software/base/src/main/java/org/apache/brooklyn/entity/software/base/SoftwareProcess.java

```diff
@@ -147,6 +147,9 @@
     @SetFromFlag("runDir")
     AttributeSensorAndConfigKey<String,String> RUN_DIR = BrooklynConfigKeys.RUN_DIR;
 
+    ConfigKey<Boolean> ADD_OPEN_PORTS_EFFECTOR = ConfigKeys.newBooleanConfigKey("effector.add.openPorts",
```

Comment:
You are using "inbound" and "open" interchangeably. Better stick to one of them for consistency.